python/editdistpy/__init__.py

```
@@ -0,0 +1,18 @@
+"""Fast Levenshtein and Damerau optimal string alignment algorithms."""
+
+from typing import Callable
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)